extensions/amp-next-page/0.2/page.js

```diff
@@ -112,14 +121,17 @@ export class Page {
   /**
    * @param {VisibilityState} visibilityState
    */
-  setVisible(visibilityState) {
+  setVisibility(visibilityState) {
     // TODO(wassgha): Handle history manipulation
     // TODO(wassgha): Handle manual visibility management
 
     // Update visibility internally and at the shadow doc level
-    if (this.shadowDoc_ && visibilityState != this.visibilityState_) {
-      this.shadowDoc_.setVisibilityState(visibilityState);
+    if (visibilityState != this.visibilityState_) {
+      this.shadowDoc_ && this.shadowDoc_.setVisibilityState(visibilityState);
```

Comment:
This is a bit of a weird pattern, would prefer to just use a normal ifs-statement.

Reply:
Would prefer an early return at the start of this function to avoid this nesting here and below. That is:

```javascript
if (visibilityState === this.visibilityState) {
  return;
}

if (this.shadowDoc_) {
  this.shadowDoc_.setVisibilityState(visibilityState);
}
```